chapters/parallel.rst

```diff
@@ -0,0 +1,157 @@
+Parallelization
+---------------
+
+Learn how Jina handles parallelization of data processing
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+Feature description and expected outcome
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Jina allows for parallelization of Peas and Pods. This generally leads
+to an increase in throughput, as the data is split across processes.
+
+Before you start
+~~~~~~~~~~~~~~~~
+
+You are expected to be familiar with the general concepts of Jina, as
+presented in
+`here <https://docs.jina.ai/chapters/core/introduction/index.html>`__.
+
+You should also have experience with parallelism in other systems. Jina
+does follow the best practices and designs from other databases or
+search engines. However, Jina is primarily a *neural search engine*.
+This means some changes and adaptations were necessary. This chapter
+will explain these.
+
+Implementation
+~~~~~~~~~~~~~~
+
+We will approach this from a top-down perspective.
+
+Inter-parallelism
+^^^^^^^^^^^^^^^^^
+
+``Pods`` are the highest level of abstraction where we can have
+parallelism. This is achieved by using the ``needs`` parameter.
+
+.. code:: python
+
+   f = (Flow().add(name='p1', needs='gateway')
+              .add(name='p2', needs='gateway')
+              .add(name='p3', needs='gateway')
+              .needs(['p1','p2', 'p3'], name='r1').plot())
+
+The plot of the above topology looks like this:
+
+|image0|
+
+In the above, ``p1``, ``p2``, ``p3`` now subscribe to ``gateway`` and
+conduct their work in parallel. The last ``.needs()`` blocks all Pods
+until they finish their work.
+
+Replicas of Pods
```

Comment:
**Suggestion**: The advantages of using this approach over the above one would be good to know...